cognite/powerops/resync/models/_shared_v1_v2/cogshop_model.py

```diff
@@ -125,16 +126,113 @@ def _create_transformation(order: int, transformation: dict | config.Transformat
+        gate_or_plant = "gate" if "gate_name" in transformation.kwargs.keys() else "plant"
+        transformation_dict = {
+            "AddWaterInTransit": {
+                "input": {
+                    "discharge_ts_external_id": transformation.kwargs.get("external_id"),
+                    "transit_object_type": gate_or_plant,
+                    "transit_object_name": transformation.kwargs.get(f"{gate_or_plant}_name"),
+                }
+            }
+        }
+    elif transformation.transformation.name == "ADD_WATER_IN_TRANSIT":
+        gate_or_plant = "gate" if "gate_name" in transformation.kwargs.keys() else "plant"
+        transformation_dict = {
+            "AddWaterInTransit": {
+                "input": {
+                    "discharge_ts_external_id": transformation.kwargs.get("external_id"),
+                    "transit_object_type": gate_or_plant,
+                    "transit_object_name": transformation.kwargs.get(f"{gate_or_plant}_name"),
+                }
+            }
+        }
+
+    elif transformation.transformation.name == "MULTIPLY":
+        transformation_dict = {"MultiplyConstant": {"input": {"constant": transformation.kwargs.get("value")}}}
+    elif transformation.transformation.name == "MULTIPLY_FROM_OFFSET":
+        transformation_dict = {
+            "MultiplyFromOffset": {
+                "input": {
+                    "relative_datapoints": [
+                        {"offset_minute": k, "offset_value": v} for k, v in transformation.kwargs.items()
+                    ]
+                }
+            }
+        }
+    elif transformation.transformation.name == "TO_BOOL":
+        transformation_dict = {"ToBool": None}
+    elif transformation.transformation.name == "TO_INT":
+        transformation_dict = {"ToInt": None}
+    elif transformation.transformation.name in [
+        "ZERO_IF_NOT_ONE",
+        "GATE_SCHEDULE_FLAG_VALUE_MAPPING",
+        "GENERATOR_PRODUCTION_SCHEDULE_FLAG_VALUE_MAPPING",
+        "PLANT_PRODUCTION_SCHEDULE_FLAG_VALUE_MAPPING",
+    ]:
+        transformation_dict = {"ZeroIfNotOne": None}
+    elif transformation.transformation.name == "ONE_IF_TWO":
+        transformation_dict = {"OneIfTwo": None}
+    elif transformation.transformation.name in ["STATIC", "DYNAMIC_STATIC"]:
+        transformation_dict = {
+            "StaticValues": {
+                "input": {
+                    "relative_datapoints": [
+                        {"offset_minute": k, "offset_value": v} for k, v in transformation.kwargs.items()
+                    ]
+                }
+            }
+        }
+    elif transformation.transformation.name == "RESERVOIR_LEVEL_TO_VOLUME":
+        transformation_dict = {"HeightToVolume": {"input": {"object_type": object_type, "object_name": object_name}}}
+    elif transformation.transformation.name in ["DO_NOTHING", "GATE_OPENING_METER_TO_PERCENT"]:
+        transformation_dict = {"DoNothing": None}
```

Comment:
Lets raise an exception if the transformation is not supported. 
```suggestion
    elif transformation.transformation.name in ["DO_NOTHING", "GATE_OPENING_METER_TO_PERCENT"]:
        transformation_dict = {"DoNothing": None}
    else:
        raise NotImplementedError(f"Unknown transformation {transformation.transformation.name}")
```